pulsar-broker/src/main/java/org/apache/pulsar/broker/web/RestEndpointMetricsFilter.java

```diff
@@ -87,6 +115,17 @@ public void filter(ContainerRequestContext req) throws IOException {
         req.setProperty(REQUEST_START_TIME, System.currentTimeMillis());
     }
 
+
+    private void recordSuccess(String path, String method, long duration) {
+        Attributes attributes = Attributes.of(PATH, path, METHOD, method);
```

Comment:
If there is something worth caching perhaps it's this object, since path and method don't change for a ResourceMethod.

Reply:
it makes sense